phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -502,10 +502,17 @@ li.breadcrumbs span:first-child > a {
 
 .rtl blockquote cite {
 	/* Username/source of quoter */
-	margin-right: 20px;
+	margin-right: 0;
 	margin-left: 0;
 }
 
+.rtl blockquote cite:before, .uncited:before {
+	/* Font Awesome quote-right */
+	content: '\f10e';
+	padding-right: 0;
+	padding-left: 5px;
+}
+
```

Comment:
In this case I'm inclined to leave it in bidi.css to keep all the RTL coding together rather than keeping all the icon coding together.